backend/src/index.js

```diff
@@ -22,11 +28,9 @@ io.on( 'connection', socket => {
 
 	const timestamp = new Date();
 
-	socket.on( 'setUser', ( message, reply ) => {
-		const issueKey = `${ message.repoName }:${ message.issueId }`;
-		if ( !db.has( issueKey ) ) {
-			db.set( issueKey, new Map() );
-		}
+	socket.on( 'setUser', async ( message, reply ) => {
+		const issueKey = createIssueKey( message.repoName, message.issueId );
+		const issueUsers = JSON.parse( await fetchUsers( issueKey ) || '{}' );
```

Comment:
There is a possibility of a race condition when two users connect to the same issue at the same time. You could use `hset` instead to fix it.

Reply:
Done. Now hset is default way to saving data in Redis.